exasol_transformers_extension/udfs/model_downloader_udf.py

```diff
@@ -0,0 +1,8 @@
+
+
+class ModelDownloader:
+    def __init__(self, exa):
+        self.exa = exa
+
+    def run(self, ctx) -> None:
+        pass
```

Comment:
If I add a mock without implementation, I usually raise NotImplementedError, this ensures that tests fail for this function fail, until I have it implemented.